query-node/schema.graphql

```diff
@@ -54,6 +49,8 @@ type Membership @entity {
 
   "The type of subscription the member has purchased if any."
   subscription: BigInt
+
+  channels: [Channel!] @derivedFrom(field: "ownerMember")
```

Comment:
Any reason this couldn't be `[Channel!]!`?

Reply:
If I remember correctly, we have discussed it (quite some time ago) and decided there will be a check for an array's existence and non-emptiness (aka `member.channels && member.channels.length`) in Atlas. It can be changed easily, and an array can be required (`[Channel!]!`) on all places in the input schema. What do you think?

Reply:
Well, from my point of view it's more convenient if arrays are always defined but just empty, less code for us. It still has the same semantic meaning I'd say - no items == no array == no relationships`

Reply:
Ok, all arrays are now required to exist.